ansible_files/aws_hetrogeneous_prepare_infra_playbook.yml

```diff
@@ -1,28 +1,13 @@
-- name: Create and configure the equinix infrastructure to run heterogneous cluster tests
+- name: Create and configure cloud infrastructure for heterogeneous cluster tests
   hosts: localhost
-  vars_prompt:
-    - name: equinix_project_id
-      prompt: Equinix metal project ID in which the device(s) will be created
-      private: false
-    - name: equinix_auth_token
-      prompt: Equinix metal API token
-    - name: equinix_ssh_private_key_path
-      prompt: Path to an SSH private key allowed to connect on the created device(s)
-      private: false
-    - name: equinix_unique_id
-      prompt: Hostname prefix which will be used to identify the device(s)
-      private: false
-    - name: equinix_tf_vars_file
-      prompt: Place where the Terrafom variable file will be stored
-      private: false
-    - name: equinix_tf_state_file
-      prompt: Place where the Terrafom state file will be stored
-      private: false
-  vars:
-    terraform_equinix_workdir: "{{ [playbook_dir, '..', 'terraform_files', 'equinix-ci-machine'] | path_join | realpath }}"
-  roles:
-    - equinix/heterogeneous_create_infra # Create the required devices to test heterogeneous scenarios
-    - equinix/export_primary_device_connection_details # Export the connection details, required for the common steps in Prow
+  tasks:
+    - name: "Prepare inventory"
+      ansible.builtin.include_role:
+        name: "aws/prepare_inventory"
```

Comment:
if we use ofcir, we are agnostic to the cloud provider? So maybe we can remove references to aws?